data/src/main/kotlin/com/mashup/dorabangs/data/pagingsource/DoraPagingSource.kt

```diff
@@ -22,18 +27,30 @@ class DoraPagingSource<T : Any> (
         val page = params.key ?: INITIAL_PAGE
         val loadSize = params.loadSize
 
+        val cachedData = cachedList[doraConvertKey(page, cacheKey)]
         return try {
-            val result = apiExecutor(page)
-            val isLastPage = (!result.pagingInfo.hasNext) || (result.pagingInfo.total == 0)
+            if (cachedData != null && !needFetchUpdate) {
```

Comment:
not()이 더 읽기 조아요 ^^,, 싫으면 시집가 
```suggestion
            if (cachedData != null && needFetchUpdate.not()) {
```

Reply:
ㅋㅋㅋ반영완료~